src/store/index.js

```diff
@@ -20,5 +24,5 @@ export default createStore(
     router: routerReducer,
     toastr: toastrReducer,
   }),
-  applyMiddleware(...middleware),
+  composeWithDevTools(applyMiddleware(...middleware)),
```

Comment:
Sure, but 
```
if (process.env.NODE_ENV !== 'production') {
  middleware.push(createLogger());
}
```
will still cause the logger to end up in the production bundle, it will just not be used. The redux dev tools do not have any negative side effects in production, except for a larger bundle size but that would not be fixed with that kind of if-statement. The redux dev tools are pretty useful in production aswell and i have developed multiple big sites where they're also used in prod. Here's more info: https://medium.com/@zalmoxis/using-redux-devtools-in-production-4c5b56c5600f

Reply:
Yeah, I need to require redux-logger within the production check itself. Let me look into the size of this package to see if I want it in the production bundle. 👍